decidim-dev/lib/tasks/locale_checker.rake

```diff
@@ -18,4 +18,14 @@ namespace :decidim do
       end
     end
   end
+
+  desc "Allows a decidim installation to sanitize its available locales"
+  task sanitize_locales: :environment do
+    Decidim::Organization.find_each do |organization|
+      sanitized_locales =
+        organization.available_locales & Decidim.available_locales.map(&:to_s)
```

Comment:
Well, I'm not sure leaving the translations around could cause any problems. And they would still be there if you reenable the language... :) I can see how the opposite is problematic in the current situation though (adding a new language and every object lacking _compulsory_ translations).